src/pages/services/aem-cf-editor/api/custom-fields/index.md

```diff
@@ -67,14 +67,31 @@ function CustomField() {
+### Override rules definition API
+
+Allowed values in `getDefinitions` when registering override rules.
+
+| Field          | Type      | Required | Description                                                                                                                                                               |
+|----------------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `url`          | `string`  |  ✔️      | URL of the page to load in the iframe that will replace the original field. The URL must have the same origin as the extension declaring the rules for field replacement. |
+| `pathExp`      | `string`  |          | RegExp for "content fragment" path.                                                                                                                                       |
+| `modelPathExp` | `string`  |          | RegExp for "content fragment model" path.                                                                                                                                 |
+| `fieldTypeExp` | `string`  |          | RegExp for field type.                                                                                                                                                    |
+| `fieldNameExp` | `string`  |          | RegExp for field name.                                                                                                                                                    |
 
 At least one valid expression must be specified: `pathExp`, `modelPathExp`, `fieldTypeExp`, or `fieldNameExp`.
+
+### Host API
+
+Methods that can be called on the `conn.host.field` object during rendering logic implementation.
+
+| Method                                              | Description                                                                                                                                            |
+|-----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `getModel(): Object`                                | Return Content Fragment Model for the currently edited Content Fragment.                                                                               |
+| `getDefaultValue(): mixed`                          | Returns the initial value for the input we are replacing.                                                                                              |
+| `getValidationState(): invalid&#124;valid`          | Returns the validation state value for the input we are replacing.                                                                                     |
```

Comment:
Is it string with two possible values or a boolean?

Reply:
String
https://git.corp.adobe.com/CQ/aem-headless-ui-commons/blob/main/packages/headless-components/src/components/Canvas/Extensible/CustomField.tsx#L23